operations/alloy-mixin/dashboards/prometheus.libsonnet

```diff
@@ -87,6 +87,72 @@ local filename = 'alloy-prometheus-remote-write.json';
   local remoteWritePanels(y_offset) = [
     panel.newRow(title='prometheus.remote_write', y=y_offset),
 
+    // Remote write success rate
+    (
+      panel.new(title='Remote write success rate in $cluster', type='timeseries') +
+      panel.withUnit('percentunit') +
+      panel.withDescription(|||
+        Percentage of samples sent by prometheus.remote_write that succeeded.
+
+        Low success rates can indicate a problem with Alloy or the remote storage.
+      |||) +
+      panel.withPosition({ x: 0, y: 1 + y_offset, w: 12, h: 10 }) +
+      panel.withQueries([
+        panel.newQuery(
+          expr=|||
+            (
+                1 - 
+                (
+                    sum(rate(prometheus_remote_storage_samples_failed_total{%(instanceSelector)s, component_path=~"$component_path", component_id=~"$component", url=~"$url"}[$__rate_interval]))
+                )
+                /
+                (
+                    sum(rate(prometheus_remote_storage_samples_total{%(instanceSelector)s, component_path=~"$component_path", component_id=~"$component", url=~"$url"}[$__rate_interval]))
+                )
+            )
+          ||| % $._config,
+          legendFormat='% of samples successfully sent',
+        ),
+      ])
+    ),
+
+    // Write latency
```

Comment:
This is just moved from below. I like SR & latency to be the topline as they're main objectives.